cipher/stream.go

```diff
@@ -80,10 +80,11 @@ func (sc *streamCipher) Partial(dst, src, key []byte) {
 	// absorb cryptographic input (which may overlap with dst)
 	if key != nil {
 		nkey := ints.Min(n, len(key)) // # key bytes available
-		sc.h.Write(key[:nkey])
+		// XXX check return error ?
```

Comment:
in fact an error-check here would be very nice. But as the main method does not return any error, I'm not sure what is best to do here.

So either remove the `_, _ = ` or do some error-checking.